<?xml version="1.0"?>
<launch>
    <param name="/use_sim_time" value="false"/> 
 
  <!-- ArUco mapping -->
   <node pkg="aruco_mapping" type="aruco_mapping" name="aruco_mapping" output="screen">
    <remap from="/image_raw" to="/camera/image_raw"/>

    <param name="calibration_file" type="string" value="$(find aruco_mapping)/data/ost.ini" /> 
    <param name="num_of_markers" type="int" value="2" />
    <param name="marker_size" type="double" value="0.75"/>
    <param name="space_type" type="string" value="plane" />
    <param name="roi_allowed" type="bool" value="false" />

  </node>  



</launch>

